<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentManagementUI</class>
 <widget class="QWidget" name="ContentManagementUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>328</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>328</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="_tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string>QTabWidget::pane { 
border:none;
background-image:url(&quot;:/backgroundContent&quot;);
background-position:bottom;
background-repeat:no-repeat;
border-radius:4;
}
QTabWidget::tab-bar {
left: 20px; 
top:-1px;
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background:transparent;
border: none;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
padding: 3px 10px 4px 10px;
margin-bottom:1px;
color:white;
font-size:15px;
font-family:Tahoma;
margin-right:4px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: rgba(227,0,63,90%);
}

QTabBar::tab:!selected {
background-color:rgba(0,111,173,90%);
}

QTabBar::tab:first{
	width:140px;
}

QTabBar::tab:middle{
	width:90px;
}

QTabBar::tab:last{
	width:55px;
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_availableTab">
      <property name="styleSheet">
       <string>background:transparent;</string>
      </property>
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="_downloadedTab">
      <property name="styleSheet">
       <string>background:transparent;</string>
      </property>
      <attribute name="title">
       <string>Seite</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="_skippedTab">
      <property name="styleSheet">
       <string>background:transparent;</string>
      </property>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
